### PR TITLE
Fix small sidebar not having scrollbar

### PR DESCRIPTION
**Pull request by KaspianDev**

### CODE REVIEW

File: custom/panel_templates/Default/assets/css/custom.css [hunk 1]
```diff
@@ -351,6 +351,12 @@ body {
     .sidebar {
         width: 17rem !important;
     }
+    
+    .sidebar.toggled {
+        width: 7rem !important;
```
Comment (samerton) [inline custom/panel_templates/Default/assets/css/custom.css:356]: This seems to force the sidebar to be always open on mobile (it should be wrapped in `@media (min-width: 768px)` to only apply to bigger screens)

**Review by samerton — CHANGES_REQUESTED** [review]

**Comment by samerton** [comment]
Thanks for this - just had a look on the latest dev code on my side and the sidebar appears to have a scrollbar already?

**Comment by KaspianDev** [comment]
> Thanks for this - just had a look on the latest dev code on my side and the sidebar appears to have a scrollbar already?

Problematic on firefox

File: custom/panel_templates/Default/assets/css/custom.css (hunk 1 above)
Comment (KaspianDev) [reply custom/panel_templates/Default/assets/css/custom.css:356]: it is in media query

**Comment by Derkades** [comment]
Indeed, there was a bug where the collapsed sidebar didn't have a scrollbar in firefox

File: custom/panel_templates/Default/assets/css/custom.css (hunk 1 above)
Comment (samerton) [reply custom/panel_templates/Default/assets/css/custom.css:356]: Thanks, this is me being lazy and not checking the above lines

**Review by samerton — APPROVED** [review]

**Pull request merged**